package.json

```diff
@@ -65,12 +65,12 @@
         "fast-memoize": "~2.5.2",
         "file-saver": "2.0.5",
         "prop-types": "15.8.1",
-        "randomcolor": "~0.6.0",
-        "rc-slider": "10.1.1",
-        "react-datepicker": "4.5.0",
-        "react-modal": "~3.11.2",
-        "react-popper": "2.2.5",
-        "react-virtualized": "~9.22.0",
+        "randomcolor": "~0.6.2",
+        "rc-slider": "10.2.1",
+        "react-datepicker": "4.16.0",
+        "react-modal": "~3.16.1",
```

Comment:
Let's use exact versions everywhere. Please remove `~` prefix for all deps in `package.json`

Reply:
:+1: